test/user-script-obj.test.js

```diff
@@ -104,4 +104,15 @@ gt_one(2);
       });
     });
   });
+
+  describe('RunnableUserScript.uuid', () => {
+    it('is set into a valid version 4 UUID', () => {
+      for (let i = 0; i < 1000; i++) {
```

Comment:
This is not a good test.  It's wasteful, and not guaranteed to check the condition we want.  Maybe stub (or dependency inject) `getRandomValues` to return at least one value below 16, and instead assert that it's formatted correctly?

Reply:
OK. I changed it.